packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -879,7 +879,6 @@ void main() {
                   children: <TestSemantics>[
                     TestSemantics(
                       flags: <SemanticsFlag>[SemanticsFlag.hasImplicitScrolling],
-                      actions: <SemanticsAction>[SemanticsAction.scrollDown, SemanticsAction.scrollUp],
```

Comment:
Why is this removed?

Reply:
This is a line of reverting the original fixes. Giving one pixel content dimensions makes it scrollable.

Reply:
Ah ok, and so it serves as a regression test. Thanks for explaining!